interfaces/builtin/firewall_control.go

```diff
@@ -36,6 +36,25 @@ const firewallControlConnectedPlugAppArmor = `
 
 #include <abstractions/nameservice>
 
+# systemd-resolved (not yet included in nameservice abstraction)
```

Comment:
Upstream decided that while this is suited for the nameservice abstraction, they felt that since non-Ubuntu (and Ubuntu derivatives) don't have dbus mediation, they didn't want to have it in there yet. I did commit it to the Ubuntu archive, but it isn't available anywhere. In cases like this I try to push the change upstream (eg, apparmor, Ubuntu, etc) *and* snapd, then at some future point (series 18?) we can evaluate if we can just use the abstraction.